.github/workflows/nuget_package_push.yml

```diff
@@ -0,0 +1,31 @@
+name: Nuget Master Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  pushrelease:
+    runs-on: windows-latest
+    env:
+      ACTIONS_ALLOW_UNSECURE_COMMANDS: "true"
+    steps:
+      - uses: actions/checkout@master
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 5.0.100
+      - name: Build/Check for compile errors (dotnet build)
+        run: dotnet build --configuration Release
+      - name: Run Unit Tests (dotnet test)
+        run: dotnet test --configuration Release
+      - name: Create Package (dotnet pack)
+        run: |
+          dotnet pack --configuration Release
```

Comment:
can we read the version (without the suffix) from a git tag?
```
dotnet pack -p:PackageVersion= read git tag here
```
(example from [official docs](https://docs.microsoft.com/de-de/dotnet/core/tools/dotnet-pack#examples))